### PR TITLE
[bugfix] nvd3's API changed and broke the range filter

### PR DESCRIPTION
**Pull request by mistercrunch**
The change that altered the feature:
https://github.com/novus/nvd3/commit/706f28b71ff93e2df23d6175971deecf0092ac26

@bkyryliuk @ascott

### CODE REVIEW

File: caravel/assets/visualizations/nvd3_vis.js [hunk 1]
```diff
@@ -48,10 +48,12 @@ function nvd3Vis(slice) {
           case 'line':
             if (fd.show_brush) {
               chart = nv.models.lineWithFocusChart();
-              chart.lines2.xScale(d3.time.scale.utc());
               chart.x2Axis
               .showMaxMin(fd.x_axis_showminmax)
               .staggerLabels(false);
+              if (chart.focus) {
```
Comment (ascott) [inline caravel/assets/visualizations/nvd3_vis.js]: in what case will `chart.focus` be falsey?

**Comment by ascott** [comment]
1 q, but otherwise looks good to me.

File: caravel/assets/visualizations/nvd3_vis.js (hunk 1 above)
Comment (mistercrunch) [inline caravel/assets/visualizations/nvd3_vis.js]: From my understanding it should always be there when using `nv.models.lineWithFocusChart`, I'll remove the `if`

**Comment by mistercrunch** [comment]
Removed the if, and tested, merging now.

**Comment by ascott** [comment]
🎉

**Pull request merged**